src/slic3r/GUI/Tab.cpp

```diff
@@ -2984,6 +2984,7 @@ void TabPrinter::build_fff()
 #endif
 
         optgroup = page->new_optgroup(L("Advanced"), L"param_advanced");
+        optgroup->append_single_option_line("use_relative_e_distances");
```

Comment:
I feel we can move this option to the  extruder tab

Reply:
Not sure... because in a multi-extruder variant of the slicer, this setting would not be "per extruder". This is how the g-code is emitted for all extruders. Superslicer and PrusaSlicer both put this into "Printer settings-> General -> Advanced" sections.

Reply:
It's a good point.
I agree.